src/server/Program.cs

```diff
@@ -30,16 +31,19 @@ public static void Main(string[] args)
             var portsToListen = new []{80, MusicCastHost.DlnaHostPort, 51100};
             var urisToListen = portsToListen.ToList().Select(p => $"http://{MusicCastHost.ThisIp}:{p}");
             var uriToListenString = string.Join(";", urisToListen);
-            builder.AddCommandLine(new[] { $"server.urls={uriToListenString}" });
-            var config = builder.Build();
+        
+            var config = builder
+                .AddCommandLine(new[] { $"server.urls={uriToListenString}" })
+                .AddEnvironmentVariables(prefix: "ASPNETCORE_")
+                .Build();
 
-            var webHost1 = new WebHostBuilder().Build();
-           
-            webHost1.Start();
-            webHost1.
-                //.UseServer("Microsoft.AspNet.Server.Kestrel")
-                //.Build()
-                //.Start();
+            var host = new WebHostBuilder()
+                .UseConfiguration(config)
+                .UseKestrel()
+                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseIISIntegration()
+                .UseStartup<Startup>()
+                .Build();
 
             Console.WriteLine($"Started the server. Listing on {uriToListenString}");
```

Comment:
This log should probably say something like
`Started the server. Listing on http://192.168.1.1:80;http://192.168.1.1:51123;http://192.168.1.1:51100`

 if the `MusicCastHost.ThisIp` is working